prism-agent/service/server/src/main/scala/io/iohk/atala/connect/controller/http/ConnectionsPage.scala

```diff
@@ -41,37 +41,37 @@ object ConnectionsPage {
   object annotations {
     object contents
         extends Annotation[Seq[Connection]](
-          description = "",
+          description = "Array of resources (Connection)\nA sequence of Connection resources representing the list of connections that the paginated response contains.",
```

Comment:
The string (with interpolation) on multiple lines that has been used in this PR may be better (https://github.com/hyperledger-labs/open-enterprise-agent/pull/861/files) avoids the use of newline characters such as \n - not a must but incase you also felt as pained by the newline character as I did ;-)

Reply:
Oh yes, good point! Will fix it 👍